dashboard/app/views/projects/index.html.haml

```diff
@@ -69,6 +69,10 @@
       $scope.projectsLoaded = true;
     }).catch($scope.genericError);
 
+    $scope.projectVisible = function(project) {
+      return (!project.hidden);
```

Comment:
This does seem like something that should be an API/service level change to
me.

On Fri, Feb 20, 2015 at 11:56 AM, Josh Lory <notifications@github.com>
wrote:

> In dashboard/app/views/projects/index.html.haml
> <https://github.com/code-dot-org/code-dot-org/pull/1360#discussion_r25097946>
> :
>
> > @@ -69,6 +69,10 @@
> >        $scope.projectsLoaded = true;
> >      }).catch($scope.genericError);
> >
> > +    $scope.projectVisible = function(project) {
> > +      return (!project.hidden);
>
> Filtering on the client side, not in the API?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/1360/files#r25097946>.
>